extract.sh

```diff
@@ -40,15 +40,15 @@ extractWinIL2CPP() {
   fi
 }
 
-for i in 5.3.0f4 5.3.1f1 5.3.2f1 5.3.3f1 5.3.4f1 5.3.5f1 5.3.6f1 5.3.7f1 5.3.8f2 5.4.0f3 5.4.1f1 5.4.2f2 5.4.3f1 5.4.4f1 5.4.5f1 5.4.6f3 5.5.0f3 5.5.1f1 5.5.2f1 5.5.3f1 5.5.4f1 5.5.5f1 5.5.6f1 5.6.0f3 5.6.1f1 5.6.2f1 5.6.3f1 5.6.4f1 5.6.5f1 5.6.6f2 5.6.7f1 2017.1.0f3 2017.1.1f1 2017.1.2f1 2017.1.3f1 2017.1.4f1 2017.1.5f1 2017.2.0f3 2017.2.1f1 2017.2.2f1 2017.2.3f1 2017.2.4f1 2017.2.5f1 2017.3.0f3 2017.3.1f1 2017.4.10f1 2017.4.11f1 2017.4.12f1 2017.4.13f1 2017.4.14f1 2017.4.15f1 2017.4.16f1 2017.4.17f1 2017.4.18f1 2017.4.19f1 2017.4.1f1 2017.4.20f2 2017.4.21f1 2017.4.22f1 2017.4.23f1 2017.4.24f1 2017.4.25f1 2017.4.26f1 2017.4.27f1 2017.4.28f1 2017.4.29f1 2017.4.2f2 2017.4.30f1 2017.4.31f1 2017.4.32f1 2017.4.33f1
+for i in 5.3.0f4 5.3.1f1 5.3.2f1 5.3.3f1 5.3.4f1 5.3.5f1 5.3.6f1 5.3.7f1 5.3.8f2 5.4.0f3 5.4.1f1 5.4.2f2 5.4.3f1 5.4.4f1 5.4.5f1 5.4.6f3 5.5.0f3 5.5.1f1 5.5.2f1 5.5.3f1 5.5.4f1 5.5.5f1 5.5.6f1 5.6.0f3 5.6.1f1 5.6.2f1 5.6.3f1 5.6.4f1 5.6.5f1 5.6.6f2 5.6.7f1 2017.1.0f3 2017.1.1f1 2017.1.2f1 2017.1.3f1 2017.1.4f1 2017.1.5f1 2017.2.0f3 2017.2.1f1 2017.2.2f1 2017.2.3f1 2017.2.4f1 2017.2.5f1 2017.3.0f3 2017.3.1f1 2017.4.10f1 2017.4.11f1 2017.4.12f1 2017.4.13f1 2017.4.14f1 2017.4.15f1 2017.4.16f1 2017.4.17f1 2017.4.18f1 2017.4.19f1 2017.4.1f1 2017.4.20f2 2017.4.21f1 2017.4.22f1 2017.4.23f1 2017.4.24f1 2017.4.25f1 2017.4.26f1 2017.4.27f1 2017.4.28f1 2017.4.29f1 2017.4.2f2 2017.4.30f1 2017.4.31f1 2017.4.32f1 2017.4.33f1 2017.4.3f1 2017.4.4f1 2017.4.5f1 2017.4.6f1 2017.4.7f1 2017.4.8f1 2017.4.9f1
```

Comment:
I should really have sorted these numerically :-\

Reply:
Yes, yes you should have 😂